src/sentry/static/sentry/app/utils.jsx

```diff
@@ -232,6 +234,18 @@ export function getShortVersion(version) {
   return version;
 }
 
+export function parseBitBucketRepo(repo) {
+  if (repo) {
+    let re = /bitbucket\.org\/([^\/]+\/[^\/]+)/i;
+    let match = repo.match(re);
+    let parsedRepo;
+    match ? (parsedRepo = match[1]) : (parsedRepo = repo);
+    return parsedRepo;
+  } else {
```

Comment:
also i think it reads easier if the function starts with a guard statement like `if( not repo) return repo`